src/Components/FacetBreadcrumbs.tsx

```diff
@@ -41,6 +41,36 @@ export default class FacetBreadcrumbs extends React.Component {
     this.setState(this.headlessBreadcrumbManager.state);
   }
 
+  getDateFacetBreadcrumbs() {
+    let dateBreadcrumbs = this.state.dateFacetBreadcrumbs;
+    return dateBreadcrumbs.map((dateBreadcrumb) => (
+      <div key={dateBreadcrumb.field}>
+        {dateBreadcrumb.values.map((value) => (
+          <div key={dateBreadcrumb.field + value.value.start}>
+            <Link
```

Comment:
Should we add the facet title like we did for regular facets?

Reply:
I thought about it but it seemed a bit too obvious, like it's just wasting space 🤔